src/main/java/org/jenkinsci/plugins/ghprb/upstream/GhprbUpstreamStatusListener.java

```diff
@@ -59,15 +60,15 @@ private boolean updateEnvironmentVars(AbstractBuild<?, ?> build, TaskListener li
             statusMessages.add(new GhprbBuildResultMessage(state, message));
         }
         
-        String context = envVars.get("commitStatusContext");
+        String context = envVars.get("ghprbCommitStatusContext");
         
         if (StringUtils.isEmpty(context)) {
             context = jobName;
         }
         
-        Boolean addTestResults = new Boolean(envVars.get("ghprbStartedStatus"));
+        Boolean addTestResults = Boolean.valueOf(envVars.get("ghprbStartedStatus"));
 
-        statusUpdater = new GhprbSimpleStatus(envVars.get("ghprbCommitStatusContext"), envVars.get("ghprbStatusUrl"), envVars.get("ghprbTriggeredStatus"), envVars.get("ghprbStartedStatus"), addTestResults, statusMessages);
+        statusUpdater = new GhprbSimpleStatus(context, envVars.get("ghprbStatusUrl"), envVars.get("ghprbTriggeredStatus"), envVars.get("ghprbStartedStatus"), addTestResults, statusMessages);
```

Comment:
Not sure I understand this suggestion.  How can you remove 63 -> 67 and still pass the job name as the context if the context is empty?  Without some really ugly inline if statement.

`new GhprbSimpleStatus(StringUtils.isEmpty(envVars.get("ghprbCommitStatusContext")) ? envVars.get("ghprbCommitStatusContext") : jobName, envVars.get("ghprbStatusUrl"), etc...`